backend/protos/mgmt/v1alpha1/anonymization.proto

```diff
@@ -7,10 +7,16 @@ import "mgmt/v1alpha1/transformer.proto";
 
 message AnonymizeManyRequest {
   // Array of stringified JSON data to be anonymized (up to 25 items)
-  repeated string input_data = 1 [(buf.validate.field).repeated = {max_items: 25}];
+  repeated string input_data = 1 [(buf.validate.field).repeated = {
+    min_items: 1
+    max_items: 25
+    items: {
+      string: {len: 1}
```

Comment:
is this right? For string primitives we defined it as `(buf.validate.field).string.min_len = 1`

this isn't requiring that everything is length 1? :)